server/views/pages/bookedtoday/arrivals/searchforexisting/noExistingRecordsFound.njk

```diff
@@ -23,7 +23,7 @@
             </div> 
             <div> 
                 {{ govukInsetText({
-                html: 'If you think this person has an existing prisoner record, you can <a class="govuk-link--no-visited-state" data-qa="search-instead" href="/manually-confirm-arrival/search-for-existing-record">search again using different details</a>.'
+                html: 'If you think this person has an existing prisoner record, you can <a class="govuk-link--no-visited-state" data-qa="search-instead" href="/prisoners/' + data.id + '/search-for-existing-record">search again using different details</a>.'
```

Comment:
Does this need to be the `"/prisoners/' + data.id + '/search-for-existing-record/new"` endpoint to clear out search details and reset them as the original search details?